client/group/group.js

```diff
@@ -0,0 +1,30 @@
+/* @flow */
```

Comment:
This whole file can actually be removed, unless you had some vision for it in the future, but I think the Groups table accomplishes most of what we'd want this for.